nginx/nginx.conf.template

```diff
@@ -26,13 +26,13 @@ http {
     open_file_cache_valid    2m;
     open_file_cache_min_uses 1;
     open_file_cache_errors   on;
-    
+
     location /s3/avalon/ {
       internal;
       proxy_pass http://$AVALON_STREAMING_BUCKET.s3.amazonaws.com/;
     }
 
-    location ~ /avalon/(?<stream>.+)/(?<resource>.+\.(?:m3u8|ts)) {
+    location ~ ^/avalon/(?<stream>.+/(?<resource>.+\.(?:m3u8|ts))) {
```

Comment:
The changes in this regex were twofold and shouldn't have really changed functionality but were made to make the regex more explicitly match what was happening.
1. Match the whole path to be more explicit and avoid leaving nginx to resolve collisions between the `/s3/avalon/` location and this location.
2. Have the "stream" capture group include the full path (instead of just the directory path) since this is what Avalon expected and was getting passed anyways.